src/libraries/System.Reflection.Context/src/System/Reflection/Context/ThrowHelper.cs

```diff
@@ -0,0 +1,21 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+// This file defines an internal static class used to throw exceptions in the
+// the System.Reflection.Context code.
+
+using System.Reflection;
+
+namespace System
+{
+    internal static class ThrowHelper
+    {
+        internal static AmbiguousMatchException GetAmbiguousMatchException(MemberInfo memberInfo)
```

Comment:
Let's not create a new type for just one throw case, also [this project is dead](https://github.com/dotnet/runtime/blob/main/src/libraries/System.Reflection.Context/README.md), probably no need to even update the exception message

Reply:
Would you prefer that my final push just leave out this entire assembly?

Reply:
Left it in, but inlined the code instead of creating a ThrowHelper